lib/email-validation/config.rb

```diff
@@ -12,6 +14,8 @@ def initialize
       @stoplight_threshold = 10
       @timeout = 60
       @after_error_hook = ->(e) {}
+      @bounced_or_blacklisted_email_message = "Your email address is blocked due to undeliverable emails. Please <small><a href='/contact' title='Customer Service'>contact customer service</a></small> if you need further assistance."
```

Comment:
Why is this in config? It doesn't look like a configuration variable - rather should be a constant.